aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,20 +1,20 @@
 ---
-remote_endpoint: https://fullnode.mainnet.aptoslabs.com
+remote_endpoint: https://fullnode.testnet.aptoslabs.com
```

Comment:
Testnet release is automated. Does anyone know if our tooling actually cares about `remote_endpoint` in that process?